bbc/simorgh
# Add Korean translations for 500 and 404 pages

Resolves https://github.com/bbc/simorgh/issues/3287

**Overall change:** Adds Korean translations for 500 and 404 pages.

**Code changes:**

- Added 500 and 404 translations to `src/app/lib/config/services/korean.js`

---


**Testing notes**

To observe the 404 error page
- go to http://localhost.bbc.com:7080/afaanoromoo/bbc_korean_radio/blah

To observe the 500 error page
- add `status = 500` here https://github.com/bbc/simorgh/blob/404Handling/src/app/routes/getInitialData/utils/fetchData/index.js#L16
- go to http://localhost.bbc.com:7080/korean

Translations can be verified here https://docs.google.com/spreadsheets/d/1IE6dvHfOJhYTIhC0kg8aRGnaj3fEg1NH9jQwo5NuUDQ/edit#gid=0
